aea/aea_builder.py

```diff
@@ -244,6 +244,7 @@ def __init__(self, with_default_packages: bool = True):
         self._context_namespace = {}  # type: Dict[str, Any]
 
         self._package_dependency_manager = _DependenciesManager()
+        self._connections: List[Component] = []
```

Comment:
why not `List[Connection]`?

Reply:
will fix with cast, cause passed component and depending on it's type it's added as resource or connection

Reply:
done